prediction.py

```diff
@@ -0,0 +1,129 @@
+
+##1. Load the libraries
+from joblib import load
+import pandas as pd
+
+##2. Use load to load both the model and the converter
+loaded_model = load("ModelDeployment_pki_file.pkl") #The line others will use when they want to load your file 
+loaded_scaler = load("scalar.pkl") #The line others will use when they want to load your file 
+
+
+##3. Bringing the new data they need to test for: gender, age, hypertension, heart_disease, ever_married, work_type, Residence_type, avg_glucose_level, bmi, smoking_status
+
+###Input #1 "gender" ---> Enter Either 'Male' or 'Female' as a string
+gender = 'Female'
+###DO NOT CHANGE ANYTHING IN THE IF STATEMENTS
+if gender == 'Male': 
+    gender_male= 1

```

Comment:
You should not be creating these variables over a over again. Insatiate them as 0 at the start and then set them to 1 if necassary.

Reply:
Thank you i will consider that

Reply:
You could have used the onehot encoder that you created before instead of doing all this.